### PR TITLE
API-7139 Fixed terms of use history page

### PR DESCRIPTION
**Pull request by petekirby-ee**

### CODE REVIEW

File: app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/AbstractApplicationController.scala [hunk 1]
```diff
@@ -38,8 +38,9 @@ abstract class AbstractApplicationController(
     with EitherTHelper[Result]
     with ApplicationLogger {
 
-  implicit class TimestampSyntax(datetime: DateTime) {
-    def asText = datetime.toString("dd MMMM yyyy")
+  implicit class TimestampSyntax(datetime: LocalDateTime) {
+    val formatter = DateTimeFormatter.ofPattern("dd MMMM yyyy").withZone(ZoneId.systemDefault())
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/AbstractApplicationController.scala:42]: Should this be ZoneOffset.UTC ?

File: test-common/uk/gov/hmrc/apiplatform/modules/submissions/SubmissionsTestData.scala [hunk 2]
```diff
@@ -32,28 +32,28 @@ trait StatusTestDataHelper {
 
     def hasCompletelyAnsweredWith(answers: Submission.AnswersToQuestions): Submission = {
       (
-        Submission.addStatusHistory(Submission.Status.Answering(DateTime.now.withZone(DateTimeZone.UTC), true)) andThen
+        Submission.addStatusHistory(Submission.Status.Answering(LocalDateTime.now(ZoneOffset.UTC), true)) andThen
```
Comment (AndySpaven) [inline test-common/uk/gov/hmrc/apiplatform/modules/submissions/SubmissionsTestData.scala:35]: There is a trait ClockNow that allows for now with offset and truncation to millis.  Might be worth a think. Also has a fixedclock trait for testing.

**Review by AndySpaven — COMMENTED** [review]
Some comments around clocks

File: test-common/uk/gov/hmrc/apiplatform/modules/submissions/SubmissionsTestData.scala (hunk 2 above)
Comment (petekirby-ee) [reply test-common/uk/gov/hmrc/apiplatform/modules/submissions/SubmissionsTestData.scala:35]: Left as system default deliberately to display date/times (although we're only displaying dates here so it's not going to make any difference).

**Pull request merged**